### ISSUE TITLE
Implicit relative imports do not work with import hook

### ISSUE THREAD

**Issue by connect2satz** [chaquo/chaquopy#12]
I am getting below error while running script  "from xml.dom import minidom" in python console. 

```
com.chaquo.python.PyException: ImportError: No module named domreg
```

I am using chaquopy android python plugin. i have a sample program which use the dom parser so i have to use "from xml.dom import minidom" in my python script. my application is crashing because it is not able to find the module "domreg" error.

**Comment by mhsmith** [comment]
Thanks: this has been fixed in version 0.4.4 which is available now.

**Comment by connect2satz** [comment]
Thanks Smith.
